src/components/GlobalStyles.js

```diff
@@ -6,6 +6,7 @@ const useStyles = makeStyles(() => createStyles({
       boxSizing: 'border-box',
       margin: 0,
       padding: 0,
+      'font-family': 'Arial, Helvetica, sans-serif'
```

Comment:
I think `roboto` was standardized across the project. Any specific reason I'm missing for using `Arial` here?

Reply:
The old dashboard was using `segoe UI` but I switched to Arial here for aesthetic and not having to import a font while loading. For a very brief moment before the font is imported, the basic font is visible which doesn't give a good UX. Let me know if you want to go with Roboto or Segoe.